galaxy_importer/loaders/schemas/patterns/pattern.json

```diff
@@ -0,0 +1,232 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
hey @jerabekjiri ! Commenting where I don't really know much about the code. I'm assuming this is a temp file for your WIP PR as the actual schema will live in another repo. (see [active PR](https://github.com/ansible/pattern-service/pull/3))

Reply:
thanks, I wasn't aware of that, I'll update the schema file.